src/plugins/memoryvalue/README.md

```diff
@@ -0,0 +1,16 @@
+**Memory Value Plugin**
```

Comment:
The metadata is missing. See https://github.com/ElektraInitiative/libelektra/blob/master/src/plugins/template/README.md

Also, use [markdown headings](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#headings) for the title.